Makefile.inc

```diff
@@ -88,6 +89,16 @@ COREBOOT_DEPS_FLAG := $(PLATFORM_BUILD_DIR)/.coreboot-deps
 COREBOOT_PATCH_FLAG := $(PLATFORM_BUILD_DIR)/.coreboot-patch
 COREBOOT_TOOLCHAIN_FLAG := $(PLATFORM_BUILD_DIR)/.coreboot-toolchain
 COREBOOT_TOOLCHAIN_CACHE_ENABLE ?= 1
+# coreboot Intel blobs
+ifdef COREBOOT_BLOBS_DIR
+COREBOOT_BLOBS_DIR := $(realpath $(COREBOOT_BLOBS_DIR))
+COREBOOT_FSP_T := $(COREBOOT_BLOBS_DIR)/Server_T.fd
```

Comment:
these are single-use vars, we don't really need them and can just do

```
sed -i "s|CONFIG_FSP_T_FILE=.*|CONFIG_FSP_T_FILE=\"$(COREBOOT_BLOBS_DIR)/Server_T.fd\"|" $@; \
```

Reply:
Done.